### DISCUSSION TITLE
How to use new face ip adapters?

### DISCUSSION THREAD

**Discussion by moonshinegloss** [lllyasviel/Fooocus#834] [Q&A]
I saw fooocus named models alongside the recently pushed: https://huggingface.co/lllyasviel/misc/blob/main/ip-adapter-plus-face_sdxl_vit-h.bin does fooocus support the face adapters yet? and if so how? the image prompt tab doesn't seem to have an option for it (yet?), thanks!

**Comment by barepixels** [comment]
Maybe lllyasviel is working on it.  If it is released lllyasviel would have announced it.

**Comment by mirek190** [comment]
WOW ip face adapter for SDXL !!

I can't wait

**Reply by ycyy** [reply]
[https://github.com/lllyasviel/Fooocus/blob/main/update_log.md#21790](2.1.790) it's ok now!!!

**Reply by moonshinegloss** [reply]
that's not the same as IP adapter, that's just faceswap

**Comment by ispulkit** [comment]
is this coming up soon? will have to move back to comfy otherwise

**Reply by mirek190** [reply]
it is already from some time

**Reply by thhung** [reply]
Can you show me how to do this? can't find on the doc btw.

**Reply by KrasimirIvanov1991** [reply]
Hey guys, did you get to know how this can be used in fooocus?
